lib/instrumentation/transaction.js

```diff
@@ -172,7 +175,7 @@ Transaction.prototype.toJSON = function () {
     sampled: this.sampled,
     context: undefined,
     span_count: {
-      started: this._builtSpans
+      started: this._builtSpans - this._droppedSpans
```

Comment:
The `_builtSpans` property now tracks all span objects that were created.  To se the number of spans started we subtract the number dropped. (see above)